Integrations/integration-AWS-S3.yml

```diff
@@ -0,0 +1,289 @@
+description: AWS - amazon public cloud , S3 service
+detaileddescription: |-
+  Secret Access Key and Access Key ID can be obtained from IAM page : https://console.aws.amazon.com/iam/.
+  Will work only with S3 service.
+configuration:
+- display: Access Key ID
+  name: accesskey
+  defaultvalue: ""
+  type: 0
+  required: true
+- display: Secret Access Key
+  name: secretkey
+  defaultvalue: ""
+  type: 4
+  required: true
+script:
+  script: |
+    import boto3
+    import io
+    import math
+    from datetime import datetime
+
+    AWS_ACCESS_KEY = demisto.params()['accesskey']
+    AWS_SECRET_KEY = demisto.params()['secretkey']
+
+    s3 = boto3.client('s3',aws_access_key_id=AWS_ACCESS_KEY, aws_secret_access_key=AWS_SECRET_KEY)
+
+    def convert_size(size_bytes):
+       if size_bytes == 0:
+           return "0B"
+       size_name = ("B", "KB", "MB", "GB", "TB", "PB", "EB", "ZB", "YB")
+       i = int(math.floor(math.log(size_bytes, 1024)))
+       p = math.pow(1024, i)
+       s = round(size_bytes / p, 2)
+       return "%s %s" % (s, size_name[i])
+
+    def create_entry(name, data, ec):
+        return {
+            'ContentsFormat': formats['json'],
+            'Type': entryTypes['note'],
+            'Contents': data,
+            'ReadableContentsFormat': formats['markdown'],
+            'HumanReadable': tableToMarkdown(name, data) if data else 'No result were found',
+            'EntryContext': ec
+        }
+
+
+    def create_bucket(args):
+        data = []
+        response = s3.create_bucket(
+                        ACL=args.get('acl'),
+                        Bucket=args.get('bucketName'),
+                        CreateBucketConfiguration={
+                            'LocationConstraint': args.get('awsRegion')
+                        })
+        data.append ({
+            'BucketName': args.get('bucketName'),
+            'AWSRegion': args.get('awsRegion')
+        })
+
+        ec = {'AWS.S3.NewBucket': data}
+        return create_entry('Created S3 Bucket', data, ec)
+
+    def delete_bucket(args):
+        data =[]
+        response = s3.delete_bucket(Bucket=args.get('bucketName'))
+        data.append ({
+            'BucketName': bucket
+            })
+        ec = {'AWS.S3.DeletedBuckets': data}
```

Comment:
I don't think that there is a need for `DeletedBuckets` context, but I think we should delete that bucket from `Bucket` list if exists there

Reply:
gonna need help with deleting an object from context